SingularityUI/app/components/requestDetail/ActiveTasksTable.jsx

```diff
@@ -45,12 +45,34 @@ const ActiveTasksTable = ({request, requestId, tasksAPI, healthyTaskIds, cleanin
 
   const tasksWithHealth = _.map(tasks, (task) => {
```

Comment:
could move this to mapStateToProps for cleanliness, but given that tasks might still be loading there I think it's fine to leave this as is